app-backend/batch/src/main/scala/groundwork/CreateTaskGrid.scala

```diff
@@ -64,11 +64,7 @@ class CreateTaskGrid(
       _ <- OptionT.liftF {
         AnnotationProjectDao
           .update(
-            annotationProject.copy(
-              aoi = footprint,
-              ready = true,
-              taskSizeMeters = Some(taskSizeMeters)
-            ),
+            annotationProject.copy(ready = true),
```

Comment:
A note: I made this change since I believe the `aoi` and `taskSizeMeters` fields' updates are already covered by the previous call to `TaskDao. insertTasksByGrid(...)`

Reply:
I think this is true